css/style.css

```diff
@@ -0,0 +1,1600 @@
+/*Обнуление*/
+*{padding:0;margin:0;border:0;}
+*,*:before,*:after{-moz-box-sizing:border-box;-webkit-box-sizing:border-box;box-sizing:border-box;}
+:focus,:active{outline:none;}
+a:focus,a:active{outline:none;}
+nav,footer,header,aside{display:block;}
+html,body{height:100%;width:100%;font-size:100%;line-height:1;font-size:14px;-ms-text-size-adjust:100%;-moz-text-size-adjust:100%;-webkit-text-size-adjust:100%;}
+input,button,textarea{font-family:inherit;}
+input::-ms-clear{display:none;}
```

Comment:
Codacy found an issue: [Expected single space before "}" of a single-line block (block-closing-brace-space-before)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)